public/styles.css

```diff
@@ -450,6 +450,10 @@ select {
     cursor: pointer;
 }
 
+.asset-list tbody tr.focus-like {
+    background-color: rgb(225, 225, 234);
```

Comment:
I could not find the style used for focused tr, thus I simply create something ....